router/router.nginx.conf.in

```diff
@@ -75,8 +75,8 @@ server {
         include /etc/nginx/proxy.conf;
     }
 
-    listen 80;
-    listen [::]:80;
+    listen 443;
```

Comment:
Can you put this in a different PR?

Reply:
huh, sorry, not sure how that got in here.